redesign.user.js

```diff
@@ -4225,6 +4262,148 @@ if (getSetting('general.prideLogo')) {
                 filter: saturate(2) !important;
             }
         `);
+
+        // set the flag style for the chosen setting
+        switch (getSetting('general.prideLogo')) {
+            case 'default':
+                GM_addStyle(css`
+                    .navbar.fixed-top .navbar-brand .logo,
+                    #logoimage {
+                        background-image: linear-gradient(
+                            #fe0000 24.7%,
+                            #fd8c00 24.7%,
+                            37.35%,
+                            #ffd000 37.35%,
+                            50%,
+                            #119f0b 50%,
+                            62.65%,
+                            #457cdf 62.65%,
+                            75.3%,
+                            #c22edc 75.3%
+                        );
```

Comment:
Ich weiß, dass das hier ursprünglich auf meinen Mist gewachesen ist, aber weil ich das bei den anderen Flaggen jetzt so formatiert habe, würde ich das auch hier empfehlen:
```suggestion
                        background-image: linear-gradient(
                            #fe0000 24.7%,
                            #fd8c00 24.7% 37.35%,
                            #ffd000 37.35% 50%,
                            #119f0b 50% 62.65%,
                            #457cdf 62.65% 75.3%,
                            #c22edc 75.3%
                        );
```